parm/FV3.input.yml

```diff
@@ -6,28 +6,6 @@
 # to obtain the namelist for each physics suite that the SRW App can 
 # run with.
 
-
```

Comment:
Is there a reason we are fully removing the code that allows the use of this suite? In the past when we have stopped officially supporting suites we have still kept the logic that allows their use (see, for example, `FV3_GFS_v15p2`).

Reply:
Good point! If we keep functionality for other suites, no longer officially supported, RRFS_v1beta functionality could remain as well.

Reply:
Updated the code, reinstated the FV3_RRFS_v1beta suite, but removed the gfs_physics_nml namelist label that was being reused for some other suites. Among others, this namelist included Noah options for land-use scheme:
    lsm: 2
    lsoil_lsm: 4
that had to be overwritten by other suites's lsm and lsoil_lsm flags for the use of RUC_LSM.

Reply:
@natalie-perlin If FV3_RRFS_v1beta isn't being removed you'll also need to restore the files `diag_table.FV3_RRFS_v1beta` and `field_table.FV3_RRFS_v1beta`, as well as these locations where it is removed from the list of options:

 - https://github.com/ufs-community/ufs-srweather-app/pull/1201/files#diff-f705e74f5f727eeb13941d863a90d850be03ad4008af3bf3bcc9fed099b4c894L33
 - https://github.com/ufs-community/ufs-srweather-app/pull/1201/files#diff-07c7b729a743994e24af097288775d509b246ee691b31820b1fd982e002488ffL960
 - https://github.com/ufs-community/ufs-srweather-app/pull/1201/files#diff-7f2900554a81ce71eac973d8d57bd64277fc05bae7978aee4d9eeb20b045e23eL36